packages/ui-kit/src/components/button/button.tsx

```diff
@@ -0,0 +1,19 @@
+import style from './style.css';
+import * as React from 'react';
+
+interface ButtonProps {
+    onClick: (event: React.SyntheticEvent) => void;
+    children: React.ReactNode;
+}
+
+export class Button extends React.PureComponent<ButtonProps> {
+    render() {
+        return (
+            <button
+                className={style.button}
+                onClick={this.props.onClick}>
+                {this.props.children}
```

Comment:
consider use destructuring for props

Reply:
Please, never use props destructuring, it makes code hard to support in feature